.github/workflows/build-from-empty-opam.yaml

```
@@ -0,0 +1,31 @@
+name: Build from empty opam switch
```

Maybe add a comment on why we need this workflow, explaining the example
we had with uri and the fact that some errors are hidden because of our use of ocaml-layer.

---

done in [1bd49b1](https://github.com/returntocorp/semgrep/pull/7417/commits/1bd49b18b3ca9b5070b39b416fd77c3674fd3cfd)